content/source/docs/enterprise/private/aws-setup-guide.html.md

```diff
@@ -237,8 +254,11 @@ In the event of the Availability Zone hosting the main instances (EC2
 and RDS) failing, traffic must be routed to the standby instances to
 resume service.
 
--   The Load Balancer listener must be reconfigured to direct traffic to
+-   If using a Load Balancer, the listener must be reconfigured to direct traffic to
```

Comment:
Is "Load Balancer" a proper noun in this case, or a regular noun? If it's just a normal noun, it shouldn't be capitalized.

Reply:
We can drop the case since it's not referencing ELB or ALB, just the fact something is providing load balancer functionality.